.github/workflows/nodejs-ci.yml

```diff
@@ -15,6 +15,26 @@ on:
       build_command:
         type: string
         default: "npm run build"
+      run_dev_test:
+        description: "A boolen value to enable Developer tests (Unit/Integration/etc.,). Set value as `true` if tests are available in your repo code"
+        type: boolean
+        required: false
+        default: false
+      developer_test_branch:
+        description: 'JSON array of developer-provided branch names to run the tests'
```

Comment:
Doesn't have to be JSON array, just say "comma separated list of branch names"

Reply:
done